_source/_posts/2022-07-21-jhipster-ci-cd.md

```diff
@@ -0,0 +1,933 @@
+---
+layout: blog_post
+title: "Continous Integration and Delivery for JHipster Microservices"
+author: jimena-garbarino
+by: contractor
+communities: [devops,security,java]
+description: "How to set up continuous integration with CircleCI and continuous delivery with Spinnaker in a JHipster microservices architecture."
+tags: [java, ci, cd, jhipster]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness
+---
+<!--
+Intro
+What is CircleCI, CI
+What is Spinnaker, CD
+Workflow
+-->
+As requested by JHipster users, in this post we cover the basics of how to add continous integration and delivery (CI/CD) for a JHipster microservices architecture and Kubernetes as the target cloud deployment environment.
```

Comment:
I don't believe JHipster users requested this. I think it was mostly me. ;) 

I was curious to see how it worked and referenced the idea in a [previous post](https://developer.okta.com/blog/2021/06/01/kubernetes-spring-boot-jhipster#continuous-integration-and-delivery-of-jhipster-microservices).

We should probably adjust this intro and the "Learn more" section to reflect this.

Reply:
I updated the intro and learn-more.